src/routes/appRoutes.tsx

```diff
@@ -31,6 +31,7 @@ import { NoAccess } from "@/components/NoAccess.js";
 import { relativePath } from "@/utils/const.js";
```

Comment:
I am like 80% sure the relativePath is a ts 
please fix it for me lest we forget
just remove the extension